pkg/config/lifecycle_provider.go

```diff
@@ -40,6 +40,14 @@ func NewLifecycleProvider(client RegistryClient, keychainFactory registry.Keycha
 	}
 }
 
+func (l *LifecycleProvider) Digest() (string, error) {
```

Comment:
What is this Digest method for?

Reply:
We used it in a previous version and forgot to delete it.
Thanks!